liblumen_eir_interpreter/src/code.rs

```diff
@@ -7,43 +7,44 @@ use libeir_ir::Block;
 use liblumen_alloc::erts::process::code::stack::frame::Frame;
 use liblumen_alloc::erts::process::code::Result;
 use liblumen_alloc::erts::process::Process;
-use liblumen_alloc::erts::term::{Atom, Term, TypedTerm};
+use liblumen_alloc::erts::term::{Atom, Boxed, Closure, Term, TypedTerm};
 use liblumen_alloc::erts::ModuleFunctionArity;
 
 use crate::exec::CallExecutor;
 
-pub fn return_throw(arc_process: &Arc<Process>) -> Result {
-    let argument_list = arc_process.stack_pop().unwrap();
-
-    panic!("{:?}", argument_list);
-
-    //let class: Atom = class_term.try_into().unwrap();
-    //let class = match class.name() {
-    //    "EXIT" => Class::Exit,
-    //    k => unreachable!("{:?}", k),
-    //};
-
-    //let exc = Exception {
-    //    class,
-    //    reason: reason_term,
-    //    stacktrace: Some(trace_term),
-    //    file: "",
-    //    line: 0,
-    //    column: 0,
-    //};
-    //result_from_exception(arc_process, exc.into())
-}
-
-pub fn return_ok(arc_process: &Arc<Process>) -> Result {
-    let argument_list = arc_process.stack_pop().unwrap();
-
-    println!("PROCESS EXIT NORMAL WITH: {:?}", argument_list);
-
-    arc_process.return_from_call(argument_list)?;
-    Process::call_code(arc_process)
-}
+//pub fn return_throw(arc_process: &Arc<Process>) -> Result {
```

Comment:
Should this be deleted?

Reply:
Deleted these comments and replaced them with working versions.